bundles/pascal/misc/example.pas

```diff
@@ -0,0 +1,53 @@
+{ Comments }
```

Comment:
@nilnor I think this is actually some sort of bug in the styling itself. I managed to make it happen once with a Python source file. Maybe it's just that the nested comments in Pascal bring it up?